### DISCUSSION TITLE
How do I update the game?

### DISCUSSION THREAD

**Discussion by TangJames** [thijsvanloef/palworld-server-docker#72] [Q&A] [answered]
Is there a command to update the game or does it auto update?

**Comment by thijsvanloef** [comment, marked as answer]
By default, when you restart the Docker container, the dedicated server will update.

**Reply by TangJames** [reply]
Perfect thank you!

**Reply by kiet-ng** [reply]
Hello Thijs!
I found this discussion and the proposed solution doesn't work. I tried:
1. docker compose restart
2. docker compose down -> up
3. docker pull and docker up

When i list the server up in Palworld I get that it is on the 0.1.2.0 version and not on the current 0.1.3.0. Do you have any suggestions?

**Reply by kiet-ng** [reply]
Hello again!

I found the problem.. need to give the container more disk space so it could download and update the server. Hope this is helpful for someone who come across the same problem later!
